tst/regression/regression_test.py

```diff
@@ -187,6 +189,7 @@ def build_code(geometry, use_gpu=False, build_type="Release"):
     for option in configure_options:
         cmake_call.append(option)
     cmake_call.append(SOURCE_DIR)
+    print(cmake_call)
```

Comment:
We should keep this. Good debugging tool. :+1:

Reply:
Definitely wasn't added for debugging and then accidentally left (but I agree, it's good to have)

Reply:
;)